### ISSUE TITLE
Sync problem

### ISSUE THREAD

**Issue by Lagerfeldt** [va7deo/MegaSys1_A#2]
**Problem**
Sync problem/dropout

**Occurs**
Always or every time a game switches between screens, such as Rod-Land from Intro > Title screen > Demo > High score > [...]

**Core tested**
MegaSys1_2023_07_02

**Hardware used**
· Ultimate MiSTer BliSTer Pro Dual RAM
· Eizo 2456 Monitor @ 1920 x 1200 (video_mode=1920,48,32,80,1200,3,6,26,154000) via HDMI
· AOC G2790VXA Monitor VRR @ 1920 x 1080 (video_mode=8) via HDMI

**Other settings**
vsync_adjust=2
refresh_min=49
refresh_max=76

**Other relevant info**
· Core build 2023-05-19 and earlier works - no sync issues. Sync issues occur with or from MegaSys1_2023_07_02
· All other cores work without sync issues on both monitors at different resolutions and refresh rates
· Manually adjusting vsync width in the game can make the problem even worse, i.e. lose sync/signal completely

Anything I can do do test or try out solutions, let me know.

**Comment by atrac17** [comment]
Upload your ini file here for me to test. That is the first thing I'd like to try and debug. Run everything exactly how you are with the latest release.

**Comment by Lagerfeldt** [comment]
Thanks. Uploaded active ini file, zipped. Exactly as is.

[MiSTer.ini.zip](https://github.com/va7deo/MegaSys1_A/files/11955242/MiSTer.ini.zip)

**Comment by atrac17** [comment]
I was able to replicate the sync issues using the provided MiSTer.ini and current release build. Commenting the following lines sync doesn't drop with the [MegaSys1_A_20230701.rbf](https://github.com/va7deo/MegaSys1_A/blob/main/releases/MegaSys1_A_20230701.rbf) build.

```
refresh_min=49
refresh_max=76
```

I'm looking at the framework to update the changes. However, I compiled the current source in this repository and sync is stable with the provided ini file. You can either comment out VRR or use this build in the interim, I'll investigate a little further and push changes on the next release.

[MegaSys1_A_20230706.zip](https://github.com/va7deo/MegaSys1_A/files/11968206/MegaSys1_A_20230706.zip)

**Comment by Lagerfeldt** [comment]
Thanks! FYI:

**With MegaSys1 A 2023-07-01**
I tried commenting those lines out some days ago in my systematic search for a potential culprit, and it did not fix the problem here. I have tried again just now with no luck as well. To the contrary it makes the sync problem worse, i.e. it loses sync even more or for longer periods.

TL;DR: No change in my setup by commenting out the min/max refresh of the monitor.

**With MegaSys1 A 2023-07-06**
Yes, now it works. But regardless of the refresh min/max. settings - it can be active or commented out in the .ini, it works with this version of the core like the previous versions of the core.

**Comment by atrac17** [comment]
That probably has something to do with your individual setup versus mine. I can only go off of what I can replicate. If you run into this problem in the future, you could download quartus 20.1 and click the play button to generate a new rbf file.
